tools/ci_build/github/azure-pipelines/android-x86_64-crosscompile-ci-pipeline.yml

```diff
@@ -0,0 +1,15 @@
+jobs:
+- job: Android_CI
+  pool: 
+    vmImage: 'macOS-10.14'
```

Comment:
For building the android package on Mac? Why do we need to test it? Why not using Linux instead?

Reply:
It seems macos has better Android emulator support: https://github.com/microsoft/azure-pipelines-image-generation/blob/master/images/macos/macos-10.14-Readme.md, and since we are doing cross-compile via official Android NDK, macos or linux doesn't matter a lot.